src/features/cosoul/MintOrBurnButton.tsx

```diff
@@ -93,10 +99,18 @@ const MintButton = ({
         }
       );
       if (receipt) {
+        await addInteractionEvent({
```

Comment:
@sshmm I was thinking the `minted` and `burned` would be done on the backend not the FE. Maybe start with adding those events to our existing handler.